Userland/Utilities/cal.cpp

```diff
@@ -201,26 +191,37 @@ ErrorOr<int> serenity_main(Main::Arguments arguments)
     if (!month && year)
         year_mode = true;
 
+    if (three_month_mode && year_mode) {
+        warnln("cal: Cannot use --three-month-view when displaying a year.");
+        return 1;
```

Comment:
Is terminating the program the right choice? how about just a little warning? (`year_mode` will override `three_month_mode`)

Reply:
I'd say so. It doesn't make sense to request both, so terminating is fine.